vimrc

```diff
@@ -238,6 +238,10 @@ let g:vim_markdown_folding_disabled = 1
 let g:go_fmt_command = "goimports"
 let g:go_highlight_trailing_whitespace_error = 0
 
+" Set Rust autocomplete trigger to control-space
+inoremap <C-@> <C-x><C-o>
```

Comment:
If you wanted to limit this to rust files, you could probably do something like:

```
autocmd FileType rust inoremap <C-@> <C-x><C-o>
```

(Like the java specific stuff, https://github.com/braintreeps/vim_dotfiles/blob/master/vimrc#L271).


I do get that c-x c-o isn't rust specific, and I have no hard feelings that it should be limited to rust files. Just raising it as a possibility.

Reply:
Ooo I like this. I think it is worth incorporating